src/pages/Admin/Dashboard.tsx

```diff
@@ -16,10 +18,19 @@ const InformationBlock = styled.div`
 
 export default function Dashboard() {
     const { t } = useTranslation('admin')
+    const history = useHistory()
+    const startPremiumVideoCall = () => {
+        history.push('/premium-video/room/testRoom?passcode=admin')
```

Comment:
this will not last long, only the first admin will be able to use it, testRoom (roomId) will be already used after.

Reply:
exactly, this will be fixed with the real room name/id. For the moment it only allows to test a "UI flow" instead of manually changing or pasting the url